test/controllers/lines_controller_test.rb

```diff
@@ -33,4 +33,13 @@ class LinesControllerTest < ActionDispatch::IntegrationTest
       get line_path(invalid_line_id)
     end
   end
+
+  test "get stops page" do
```

Comment:
This test could be simplified in a few ways. First, you don't need to test that your fixture data is available in my mind, and you can then use it `param_line`'s definition inline, changing this to:
```ruby
    get get_stops_path(lines(:one).id)
    line = assigns(:line)
    assert_response :success
```